src/app/EventLoggingTypes.h

```diff
@@ -140,6 +141,7 @@ class EventOptions
     PriorityLevel mPriority = PriorityLevel::Invalid;
     Type mUrgent            = Type::kNotUrgent; /**< A flag denoting if the event is time sensitive.  When kUrgent is set, it causes
                                                        the event log to be flushed. */
+    DataModel::Nullable<chip::FabricIndex> mFabricIndex;
```

Comment:
This needs to very clearly document what the semantics of mFabricIndex are (esp. if it's null).

Reply:
from spec, "The fabric index of the fabric that performed the lock operation. This SHALL be null if there is no fabric that can be determined for the given operation source. This SHALL NOT be null if the operation source is "Remote".,
For null, I think it means this event can be retrieved by read handler, and there is no fabric restriction on it.

Reply:
All door lock events have no fabric restrictions.

Reply:
Per offline chat, we would check for a field that is of type fabric_idx and not nullable which could be used for fabric-scoped event, currently acl is fabric scope one. updated the code